### PR TITLE
fix(settings): user badge improvements

### PR DESCRIPTION
**Pull request by Chrissy**
* fix link
* add spacing between elements
* style email test
* use it for org members list!

image

image

improved look for project settings search:

image

image

### CODE REVIEW

**Review by ckj — APPROVED** [review]
Nice polish here. Looks way better.

File: src/sentry/static/sentry/app/views/settings/organization/members/organizationMemberRow.jsx [hunk 1]
```diff
@@ -96,22 +85,12 @@ export default class OrganizationMemberRow extends React.PureComponent {
     // member has a `user` property if they are registered with sentry
     // i.e. has accepted an invite to join org
     let has2fa = user && user.has2fa;
-    let detailsUrl = recreateRoute(id, {routes, params});
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/organization/members/organizationMemberRow.jsx]: this seems to work just fine using the `Link` component built into `UserBadge`. Could someone (perhaps @MaxBittker or @billyvg) explain to me if this is true or not? Don't want to accidentally muck with browser history.

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/organization/members/organizationMemberRow.jsx]: Did you test old and new settings?

Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/organization/members/organizationMemberRow.jsx]: flip billy you are right 😭

Probably just gonna revert this whole file, as it looks ok rn and I don't really want to include all of this stuff in `UserBadge`.

**Pull request merged**